tests/api/lcp/test_controller.py

```diff
@@ -10,13 +10,13 @@
 from api.lcp.server import LCPServer
 from core.lcp.credential import LCPCredentialFactory, LCPUnhashedPassphrase
 from core.model import ExternalIntegration
-from tests.api.lcp import fixtures
-from tests.api.test_controller import ControllerTest
+from tests.api.lcp import lcp_strings
```

Comment:
I renamed this file since it is just a bunch of strings and also not to confuse with fixtures we are using now.

@io7m I decided to leave this just as strings, not to convert it to fixture that contain strings. I know there is this [block](https://www.notion.so/lyrasis/Test-Fixtures-in-the-CM-4848df744ef141d0ab48392c448f60dd?pvs=4#93e056c47a474122a279c435d3148e98) in the fixtures docs but I'm not sure there is any benefits to moving this to fixture since these are just strings, no other data manipulation or side effects. 
Let me know what you think, if you think this would be better to convert to some kind of `LCPStringsFixture` or something like that I'm open to that.

Reply:
I think it's fine to just have a module full of constants. :+1:

Reply:
The difference is that those LCP strings are constants. The example data in the documentation are actually created at runtime and so are "different" every time.